react/remote-work-demo/src/components/TeamChat.jsx

```diff
@@ -0,0 +1,131 @@
+import { useState, useCallback } from "react";
+import Talk from "talkjs";
+import { Chatbox, useSession } from "@talkjs/react";
+import talkJsConfig from "../talkJsConfig";
+import CategoryCollapse from "./CategoryCollapse";
+import ConversationListItem from "./ConversationListItem";
+import ChatHeader from "./ChatHeader";
+
+const TeamChat = ({ unreadMessages }) => {
+  const initialConversation = talkJsConfig.conversations.channels[0];
+  const [currentConversation, setCurrentConversation] =
+    useState(initialConversation);
+
+  const session = useSession();
+
+  const [mobileChannelSelected, setMobileChannelSelected] = useState(true); //This is used to control whether or not to display the chatbox or the inbox while on mobile displays
+
+  const changeConversation = (conversation) => {
+    if (session?.isAlive) {
+      const talkJsConversation = session.getOrCreateConversation(
+        conversation.id
+      );
+      const me = new Talk.User(talkJsConfig.userId);
+
+      talkJsConversation.setParticipant(me);
+      talkJsConversation.setAttributes(conversation);
+      setMobileChannelSelected(true);
+      setCurrentConversation(conversation);
+
+      const chatbox = session.getChatboxes()[0];
```

Comment:
Why not use `chatboxRef` as was the case in the previous implementation?

Reply:
(explained on Slack...)

Thanks, updated now to use `chatboxRef`.